packages/client-readmodel-writer/test/consumerService.integration.test.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable functional/immutable-data */
+/* eslint-disable functional/no-let */
+import { afterEach, afterAll, beforeAll, describe, expect, it } from "vitest";
+import {
+  ClientCollection,
+  ReadModelRepository,
+  readModelWriterConfig,
+} from "pagopa-interop-commons";
+import { mongoDBContainer } from "pagopa-interop-commons-test";
+import { StartedTestContainer } from "testcontainers";
+
+describe("Integration tests", async () => {
+  let clients: ClientCollection;
+  let startedMongoDBContainer: StartedTestContainer;
+
+  const config = readModelWriterConfig();
+
+  beforeAll(async () => {
+    startedMongoDBContainer = await mongoDBContainer(config).start();
+
+    config.readModelDbPort = startedMongoDBContainer.getMappedPort(27017);
+
+    const readModelRepository = ReadModelRepository.init(config);
+    clients = readModelRepository.clients;
+  });
+
+  afterEach(async () => {
+    await clients.deleteMany({});
+  });
+
+  afterAll(async () => {
+    await startedMongoDBContainer.stop();
+  });
```

Comment:
Can you implement the new way of initialize the test?

Reply:
Done